src/custom/state/claim/hooks/index.ts

```diff
@@ -130,15 +130,22 @@ type ClassifiedUserClaims = {
   available: UserClaims
   expired: UserClaims
   claimed: UserClaims
+  isLoading: boolean
 }
 
 /**
  * Gets all user claims, classified
- *
- * @param account
  */
 export function useClassifiedUserClaims(account: Account, optionalChainId?: SupportedChainId): ClassifiedUserClaims {
-  const userClaims = useUserClaims(account, optionalChainId)
+  const { claims: userClaims, isLoading: areClaimsLoading } = useUserClaims(account, optionalChainId)
+
+  const [isLoading, setIsLoading] = useState(false)
```

Comment:
Do you think we can set this to `true` by default?
Looks like initial `false` is causing some other to be displayed before the loader is shown https://watch.screencastify.com/v/wcpxIulZmRHPSchdiiLm

Reply:
Hmm good call, I'll try that